test/es-module/test-cjs-legacyMainResolve.js

```diff
@@ -82,7 +82,7 @@ describe('legacyMainResolve', () => {
           {},
           ''
         ),
-      { message: /instance of URL/, code: 'ERR_INVALID_ARG_TYPE' },
+      { code: 'ERR_INTERNAL_ASSERTION' },
```

Comment:
This internal error is being thrown because you're testing an internal function, a user won't get this error when using Node.js, right?

Reply:
No, unless there's a bug in Node.js